### ISSUE TITLE
Daily Email with # of crashes

### ISSUE THREAD

**Issue by cujo30227** [microsoft/appcenter#1378]
**Describe the solution you'd like**
In HockeyApp, I used to receive a daily email that summarizes the number of crashes for each of my apps from the previous day. This was very useful in monitoring any breakdowns of the apps. I would like to see this feature in AppCenter as well.

**Describe alternatives you've considered**
Email updates on new crash groups. This is not helpful in monitoring the status, though.

**Additional context**
N/A

**Comment by jerietve** [comment]
Please have a look at #853. This looks like a duplicate. Good news over there though, it's almost ready!

**Comment by stephent** [comment]
I don't think this is a duplicate - #853 (now closed) appears only to solve for new crash groups, whereas the primary use case for me (and OP) is to have your finger on the pulse of overall crash trends - did my new release break something? did an OS update break something? etc. That's possibly, but certainly not necessarily, related to new crash groups - we need a daily email that shows overall crash report trends, new and existing.

**Comment by cujo30227** [comment]
Yes, that’s exactly what I meant, and #853 indeed doesn’t address my question. 
Thanks for clarifying @stephent

**Comment by winnie** [comment]
Thanks for feedback! This is a common scenario we've heard before and we're currently exploring ways to make it easier for users to keep up to date on their app trends (via some dashboard or email). We don't have concrete plans yet but will post here with any updates. Thanks!

**Comment by cujo30227** [comment]
Thanks! Unfortunately, for me, the transition from Hockeyapp to Appcenter has decreased the Value-Add enormously. In the past, I would receive a daily email with the number of crashes all my apps had, and I was able to quickly spot problems. In Appcenter, I now have to go into the dashboard of each of the of my 40+ apps to look at the crashes. Lot's of additional work, and I'm considering going to another platform when I don't see improvements soon.

**Comment by PGHM** [comment]
This would still be excellent to have, we are also considering moving to another platform (for other reasons also) but this would help a lot in the monitoring sense

**Comment by PGHM** [comment]
This is still an issue and this ticket should be closed

**Comment by stephent** [comment]
> This is still an issue and this ticket should be closed

^”should NOT be closed” is what’s intended I think, and I agree.

What is the point of “stale bot”? It’s like an electronic ostrich, sticking its head in the sand thinking “if I ignore it, I can make it go away”. Well, not really…

**Comment by cujo30227** [comment]
It is still an issue , but simply being ignored by the development team. Since almost 2 years ! This is a significant degradation compared to hockeyapp in the past, where the feature was available. Why is this ignored ???

**Comment by PGHM** [comment]
Staling this will not make it go away...

**Comment by PGHM** [comment]
Bump.

**Comment by PGHM** [comment]
Still very much a needed feature, no need to stale this out. Any report if this couls be done in the near future? @winnie

**Comment by stephent** [comment]
> Staling this will not make it go away...

Like @PGHM said.

**Comment by PGHM** [comment]
Making the compulsory comment again

**Comment by cujo30227** [comment]
Bump, still not stale. After almost 3 years we are still waiting for a solution here. What is so difficult about this? The daily email with hockeyapp was a real value add and allowed us to track for any abnormal behavior. Now the developers have to actively click through all their apps to get a status. 
microsoft , please address this !!!

**Comment by cujo30227** [comment]
Bump, still not stale. After 3 years (!!!) we are still waiting for a solution here, or at least a response. 

The daily email with hockeyapp was a real value add and allowed us to track for any abnormal behavior. Now the developers have to actively click through all their apps to get a status.

microsoft , please address this !!!

**Comment by PGHM** [comment]
Let the dance continue.

@winnie, could you please provide some kind of update of any plans?

**Comment by cujo30227** [comment]
Bump, still not stale. After 3,5 years (!!!) we are still waiting for a solution here, or at least a response.

**Comment by PGHM** [comment]
Well, we decided to give up using AppCenter, good work on not listening to the customers :)

**Comment by cujo30227** [comment]
Yeah, same here. Removed it from all our apps. Not listening to customers, and since we implemented the latest build, the Dashboard doesn't show crash information anymore, it shows an error (see attached). This has been since 1.5 weeks already, and also there, there is apparently nobody that takes care... Seems like a dead project. Such a pity that we don't have HockeyApp anymore, that worked great.

<img width="561" alt="image" src="https://user-images.githubusercontent.com/5006884/211879130-ba1da6a6-2cf3-40a2-9ac0-63e5e7d1fcd1.png">

**Comment by cujo30227** [comment]
Not stale.. 4 years and waiting for a response from Microsoft
